### ISSUE TITLE
No Batchnormalization in Discriminator Net

### ISSUE THREAD

**Issue by xnqio** [lucidrains/lightweight-gan#79]
I'm wondering why there ain't any Batch Normalization going on in the discriminator as supposed in the paper "Towards faster and stabilized GAN training..."
Is there a reason why you got rid of that normalization? Or am I just missing the correct codeline?

**Comment by lucidrains** [comment]
@xnqio it's because i found that gradient penalty (out of batch norm, spectral norm) gave me the best results https://arxiv.org/pdf/1704.00028.pdf

**Comment by lucidrains** [comment]
the stylegan2 paper also noted gradient penalty gave best results for their case, so my experience is partially corroborated

**Comment by xnqio** [comment]
Ok interesting, never heard of gradient penalty, will check it out. Thanks for your answer!
